superset-frontend/src/SqlLab/components/ExploreCtasResultsButton/index.jsx

```diff
@@ -35,29 +35,19 @@ const propTypes = {
   templateParams: PropTypes.string,
 };
 
-class ExploreCtasResultsButton extends React.PureComponent {
-  constructor(props) {
-    super(props);
-    this.visualize = this.visualize.bind(this);
-    this.onClick = this.onClick.bind(this);
-  }
-
-  onClick() {
-    this.visualize();
-  }
-
-  buildVizOptions() {
+function ExploreCtasResultsButton(props) {
+  function buildVizOptions() {
```

Comment:
Changeall of these to 
`const buildVizOptions = () =>`
to confirm with our style guide